### PR TITLE
Update Russian translation

### LINKED ISSUE #7285: Incorrect Russian translations
### Your question

I get it that Buka's work back in 90s wasn't perfect, but some of re-translations are just plain wrong.

For example, we have 2 objects: one gives +1 sp to hero permanently and another gives a secondary skill. In RU version they are named the same - "Хижина ведьмы". Also, the changes of faction's names are strange. Warlock is an "evil" faction, but now they have the part about that omitted - they are using old Wizard's name while actually they are supposed to be evil counterpart for Sorceress faction. Ogres now became "maneaters" for strange reason. 

This list can go on and on and on, but please - consider reverting some changes to Buka's original (I hope that you didn't ended up with 7Wolf translation as base for RU TL)

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by Districh-ru**
Close #7285, translate new phrases and correct some phrases.

### CODE REVIEW

**Review by ihhub — APPROVED** [review]

File: files/lang/ru.po [hunk 1]
```diff
@@ -8186,16 +8195,16 @@ msgid "Wolves"
...
-msgstr "Людоед"
+msgstr "Огр"
 
 msgid "Ogres"
-msgstr "Людоеды"
+msgstr "Огры"
 
 msgid "Ogre Lord"
-msgstr "Свирепый людоед"
+msgstr "Лорд огрв"
```
Comment (undef21) [inline files/lang/ru.po:8204]:
```suggestion
msgstr "Лорд огров"
```

Comment (Districh-ru) [reply files/lang/ru.po]: Hi, @undef21. Thanks for fixing the typo!

File: files/lang/ru.po [hunk 2]
```diff
@@ -3209,7 +3217,7 @@ msgid "File to Load:"
 msgstr "Загрузить файл:"
 
 msgid "Accept the choice made."
-msgstr "Принять изменения."
+msgstr "Принять сделанный выбор."
```
Comment (Branikolog) [inline files/lang/ru.po:3220]: Hi, @Districh-ru 
What if we use more simple, but commonly used phrase: "Подтвердить Ваш выбор"? 
"Принять сделанный выбор." - sounds like google-translation.

**Comment by ihhub** [comment]
@Districh-ru , thank you so much for this update!

**Pull request merged**